src/shadowbox/server/manager_service.ts

```diff
@@ -228,7 +229,18 @@ export class ShadowsocksManagerService {
   public createNewAccessKey(req: RequestType, res: ResponseType, next: restify.Next): void {
     try {
       logging.debug(`createNewAccessKey request ${JSON.stringify(req.params)}`);
-      this.accessKeys.createNewAccessKey().then((accessKey) => {
+      let encryptionMethod = req.params.method;
+      logging.debug(`encryptionMethod: $(encryptionMethod)`);
```

Comment:
Let's remove this, since the encryption method is already captured in the initial logging above.

Reply:
done.